src/connectors/blockchain/curation.ts

```diff
@@ -49,86 +73,98 @@ const chainId = isProd
 // CurationContract
 
 export class CurationContract extends BaseContract {
```

Comment:
Might take a while to migrate this particular contract logic to viem since it needs a major rewrite basically, apologize for the delay. Everything else should be good to go

Reply:
Thanks! We will take over this issue and keep building on top of your PR.

Reply:
@robertu7 Appreciate it! Sorry I was caught up with some family stuff over the past couple months. Glad that this PR got in :)